db/seeds.rb

```diff
@@ -5,3 +5,48 @@
 #
 #   movies = Movie.create([{ name: "Star Wars" }, { name: "Lord of the Rings" }])
 #   Character.create(name: "Luke", movie: movies.first)
+
+
+
+users = [
```

Comment:
Optional, since we won't touch this much, but keep eyes out for indentation going forward.